actionpack/lib/action_controller/renderer.rb

```diff
@@ -20,7 +20,7 @@ module ActionController
   # #render method allows you to use any options as when rendering in controller.
   # For example,
   #
-  #   FooController.render :action, locals: { ... }, assigns: { ... }
+  #   FooController.render :index, locals: { ... }, assigns: { ... }
```

Comment:
I think that makes sense actually. I think all of us were blind to the change. Maybe make this change make more sense by adding to the example `For example, to render the :index action you can:`